post-to-aws/src/main/java/no/sikt/nva/brage/migration/aws/S3StorageImpl.java

```diff
@@ -84,13 +85,20 @@ public void storeProcessedCollections(String[] collections) {
         try {
             var collectionFiles = getCollections(stripZipBundlesToCollections(collections));
             var listOfRecordsCollections = getRecordsJsonFiles(collectionFiles);
-            extractRecords(listOfRecordsCollections).forEach(this::storeRecord);
+            var records = extractRecords(listOfRecordsCollections);
+            records.forEach(this::storeRecord);
+            logSuccessfullyProceedRecords(records);
             writeLogsToS3(customer);
         } catch (Exception e) {
             logger.error(PROBLEM_PUSHING_PROCESSED_RECORDS_TO_S3 + e);
         }
     }
 
+    private static void logSuccessfullyProceedRecords(List<Record> records) {
+        logger.info(String.join(StringUtils.EMPTY_STRING, SUCCESSFULLY_PROCEEDED_MESSAGE,
+                                String.valueOf(records.size())));
+    }
```

Comment:
Adding following log message, cause for now we do not log anything when pushing records to aws only